BA_project_0911_1643.qmd

```diff
@@ -42,9 +51,11 @@ In this section, we outline the process of data collection and merging. The data
 
 The two datasets were successfully merged into a single primary data frame using the IMDb ID as the key. This process resulted in a consolidated dataset comprising 1,792 rows. Specific columns of interest were selected to facilitate further analysis and ensure the relevance of the data to our research objectives.
 
-```{r,echo=FALSE, warning=FALSE}
+```{r}
+#| code-fold:true
+#| code-summary: "Click to show code"
```

Comment:
This is blocking the html format from being knitted (R code works but rendering html no). According to the doc, it's also correct to put it in the metadata of the document
https://quarto.org/docs/output-formats/html-code.html

Reply:
I changed it and it works

Reply:
https://github.com/user-attachments/assets/9d32f2d1-7fa6-4bb3-85d4-c1924cbd7d78

Reply:
ok great so now we don't have to rewrite those 2 lines of codes in each chunk ?

How do we commit the changes into main ? Does Ivy have to do it ?

Reply:
Great, thank you!  Now, how me merge into the main with the correction you made?

Reply:
Done, the merging into main is a bit restricted to avoid accidental merges